src/data/detail-page/charts/backend.ts

```diff
@@ -0,0 +1,5 @@
+import { Chart } from './chartInterfaces';
+
+export const backend: Chart = {
+  hasChart: false,
```

Comment:
I'm not sure, if we need the property `hasChart`. Category for which we don't want to display chart, could have type set to null and data could be an empty array and it would be consistant to other data structures in our app, like for example resources.

Reply:
Ok :) Deleted.